modules/acquia_cms_tour/acquia_cms_tour.module

```diff
@@ -53,3 +53,24 @@ function acquia_cms_tour_modules_uninstalled(array $modules) {
     \Drupal::state()->delete('acms_' . $module . '_configured');
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function acquia_cms_tour_preprocess_acquia_cms_tour_checklist_form(&$variables) {
+  $showText = 0;
+  $modules = [
+    'google_analytics',
+    'geocoder',
+    'captcha',
+    'cohesion',
+    'google_tag',
+    'acquia_telemetry',
+  ];
+  foreach ($modules as $module) {
```

Comment:
@panshulK We can't hardcode this module as there could be other plugins which might provide plugin and we should show them. We should get the list from the plugins. See from here: https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_tour/src/Controller/DashboardController.php#L162

Reply:
Done!